File: integration.example/src/com/example/IntegrationExample/MainActivity.java
package com.example.IntegrationExample;

import android.os.Bundle;
import android.support.v4.app.FragmentActivity;
import android.support.v4.app.FragmentTabHost;

public class MainActivity extends FragmentActivity {
	    
	private FragmentTabHost mTabHost;
	
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
                
        mTabHost = (FragmentTabHost)findViewById(android.R.id.tabhost);
        
        mTabHost.setup(this, getSupportFragmentManager(), R.id.tabcontent);
		
		mTabHost.addTab(mTabHost.newTabSpec(getString(R.string.tab_accept)).setIndicator(getString(R.string.tab_accept)), 
				AcceptPaymentFragment.class, null);
		mTabHost.addTab(mTabHost.newTabSpec(getString(R.string.tab_reverse)).setIndicator(getString(R.string.tab_reverse)), 
				ReverseFragment.class, null);
		mTabHost.addTab(mTabHost.newTabSpec(getString(R.string.tab_printer)).setIndicator(getString(R.string.tab_printer)), 
				PrinterFragment.class, null);
    }
    


}

File: integration.example/app/src/main/java/com/example/IntegrationExample/MainActivity.kt
package com.example.IntegrationExample

import android.os.Bundle
import android.support.v4.app.FragmentActivity
import android.support.v4.app.FragmentTabHost

class MainActivity : FragmentActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)

        (findViewById(android.R.id.tabhost) as? FragmentTabHost)?.let {
            it.setup(this, supportFragmentManager, android.R.id.tabcontent)

            it.addTab(it.newTabSpec(getString(R.string.tab_accept)).setIndicator(getString(R.string.tab_accept)),
                    AcceptPaymentFragment::class.java, null)
            it.addTab(it.newTabSpec(getString(R.string.tab_reverse)).setIndicator(getString(R.string.tab_reverse)),
                    ReverseFragment::class.java, null)
            it.addTab(it.newTabSpec(getString(R.string.tab_printer)).setIndicator(getString(R.string.tab_printer)),
                    PrinterFragment::class.java, null)
        }
    }
}